ivy/functional/ivy/random.py

```diff
@@ -51,18 +51,42 @@ def random_uniform(
         If size is ``None`` (Default), a single value is returned.
     device
         device on which to create the array 'cuda:0', 'cuda:1', 'cpu' etc.
+        (Default value = None).
 
     Returns
     -------
     ret
         Drawn samples from the parameterized uniform distribution.
 
-    Examples
-    --------
-    >>> y = ivy.random_uniform(0.0, 2.0)
+    Functional Examples
+    -------------------
+    
+    >>> y = ivy.random_uniform()
     >>> print(y)
-    ivy.array(1.89150229)
+    ivy.array(0.26431865)
 
```

Comment:
Thanks for all the edits!

As this function has 4 input parameters, you need a minimum of 4 types of functional example variations.
So could you include 1 more example that takes maybe 2 parameters.
After that you should be done!